### PR TITLE
Arcbox 3.0 - Customize tags

### PR DESCRIPTION
**Pull request by sebassem**

### CODE REVIEW

File: azure_jumpstart_arcbox/artifacts/installArcAgentUbuntu.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ bash ~/install_linux_azcmagent.sh # 2>/dev/null
 ArcServerResourceName=$(hostname |sed -e "s/\b\(.\)/\u\1/g")
 
 # Run connect command
-azcmagent connect --access-token $accessToken --resource-group $resourceGroup --tenant-id $tenantId --location $Azurelocation --subscription-id $subscriptionId --resource-name "${ArcServerResourceName}" --cloud "AzureCloud" --tags "Project=jumpstart_arcbox" --correlation-id "d009f5dd-dba8-4ac7-bac9-b54ef3a6671a"
+azcmagent connect --access-token $accessToken --resource-group $resourceGroup --tenant-id $tenantId --location $Azurelocation --subscription-id $subscriptionId --resource-name "${ArcServerResourceName}" --cloud "AzureCloud" --correlation-id "d009f5dd-dba8-4ac7-bac9-b54ef3a6671a"
```
Comment (janegilring) [inline azure_jumpstart_arcbox/artifacts/installArcAgentUbuntu.sh:17]: Should the --tags parameter be specified here as well?

File: azure_jumpstart_arcbox/artifacts/installK3s.sh [hunk 2]
```diff
@@ -152,7 +152,7 @@ if [[ "$k3sControlPlane" == "true" ]]; then
     workspaceResourceId=$(sudo -u $adminUsername az resource show --resource-group $resourceGroup --name $logAnalyticsWorkspace --resource-type "Microsoft.OperationalInsights/workspaces" --query id -o tsv)
     echo "Log Analytics workspace id $workspaceResourceId"
 
-    sudo -u $adminUsername az connectedk8s connect --name $vmName --resource-group $resourceGroup --location $location --tags 'Project=jumpstart_arcbox'
+    sudo -u $adminUsername az connectedk8s connect --name $vmName --resource-group $resourceGroup --location $location
```
Comment (janegilring) [inline azure_jumpstart_arcbox/artifacts/installK3s.sh:155]: Should the --tags parameter be specified here as well?

**Review by janegilring — CHANGES_REQUESTED** [review]
Looks good, but does SQL MI and PostgreSQL resources get tags assigned?

File: azure_jumpstart_arcbox/artifacts/installArcAgentUbuntu.sh (hunk 1 above)
Comment (sebassem) [reply azure_jumpstart_arcbox/artifacts/installArcAgentUbuntu.sh:17]: The tags are applied using policy, so the machine would pick them up without setting them explicitly

File: azure_jumpstart_arcbox/artifacts/installK3s.sh (hunk 2 above)
Comment (sebassem) [reply azure_jumpstart_arcbox/artifacts/installK3s.sh:155]: The tags are applied using policy, so the machine would pick them up without setting them explicitly

**Comment by sebassem** [comment]
> Looks good, but does SQL MI and PostgreSQL resources get tags assigned?

I'm testing now if they get the tags through policy, if not then will have to add them manually using the parameter

**Comment by sebassem** [comment]
> > Looks good, but does SQL MI and PostgreSQL resources get tags assigned?
> 
> I'm testing now if they get the tags through policy, if not then will have to add them manually using the parameter

Tags are applied using the policy after testing

**Pull request merged**